perfkitbenchmarker/packages/sysbench05plus.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Module containing sysbench 0.5 (and later) installation and cleanup logic.
```

Comment:
Ah, yes indeed I have to download the source, compile, make and then install sysbench 0.5 on CentOS. What a pain. But note that at the end of the installation, I've already removed the entire source sub-directory under home directory:

 # Cleanup the source code enlisthment from bzr, we don't need it anymore.
  vm.RemoteCommand('cd ~ && rm -fr ./sysbench')

So the good news is we will have a clean home directory and left no trace behind there.

However, I do acknowledge that sysbench binaries, man files, ".lua" test scripts (and who-knows-what-else) have all been installed into various directories of the system, it will be very hard to find a laundry list of them and delete them piece by piece manually  - unless Sysbench itself provides a "make uninsall" command but I don't seem to find one.